config/prow/cluster/monitoring/mixins/prometheus/plank_alerts.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    local componentName = $._config.components.plank,
+    groups+: [
+      {
+        name: 'Heartbeat ProwJobs',
+        rules: [
+          {
+            alert: 'No recent successful runs: `%s`' % job.name,
+            expr: |||
+              increase(prowjob_state_transitions{job_name="%s", state="success"}[%s]) < 0.5
```

Comment:
Please add a comment explaining what this does.

Reply:
Done.